### DISCUSSION TITLE
Welcome to Baltica Discussions!

### DISCUSSION THREAD

**Discussion by tbrittoborges** [dieterich-lab/Baltica#13] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by onionpork** [comment]
Hi there, 

My name is Yirui. I am new to splicing world. I am involving a project that required integrating RNA splicing detection from rmats and leafcutter. This tool is exactly what I am looking for! I am looking forwards more tutorials, and I hope I can help as well.
